packages/moonstone/styles/variables.less

```diff
@@ -224,11 +224,12 @@
 
 // FormCheckbox
 // ---------------------------------------
-@moon-formcheckbox-size: (@moon-item-height * 0.8);	// Relative to `Item` height
-@moon-formcheckbox-size-large: (@moon-item-height-large * 0.8);	// Relative to `Item` height
+@moon-formcheckbox-size: (@moon-item-height * 0.55);	// Relative to `Item` height
+@moon-formcheckbox-size-large: (@moon-item-height-large * 0.55);	// Relative to `Item` height
 @moon-formcheckbox-check-icon-size: 72px;
 @moon-formcheckbox-check-icon-size-large: 84px;
-@moon-formcheckbox-border-size: 3px;
+@moon-formcheckbox-border-size: 2px;
```

Comment:
why 2px? Is this to match radio item's border?

Reply:
That's a GUI decision. It was preferable to GUI to have this be specifically 2px rather than 3 or to match radio (Radio is a circle, while form FormCheckbox is (now) a square, so the perceived thickness is more relevant than the actual.)